evm/cita-vm/Dockerfile

```diff
@@ -1,33 +1,9 @@
-FROM ubuntu:18.04
+FROM ewasm/bench-build-base:1
 
-# https://github.com/kuralabs/docker-python3-dev/blob/master/Dockerfile
-
-# System deps
-RUN apt-get update
-RUN apt-get install -y software-properties-common git sudo build-essential wget curl nano \
-    autoconf automake cmake libtool make unzip zlib1g-dev texinfo \
-    gcc musl-dev
-
-# Install Python stack
-RUN apt-get update \
-    && apt-get --yes --no-install-recommends install \
-        python3 python3-dev \
-        python3-pip python3-venv python3-wheel python3-setuptools \
-        build-essential \
-        python-dev \
-        graphviz git openssh-client \
-    && rm -rf /var/lib/apt/lists/*
-
-# install rust
-RUN curl https://sh.rustup.rs -sSf | \
-    sh -s -- --default-toolchain stable -y && . $HOME/.cargo/env
-ENV PATH=/root/.cargo/bin:$PATH
-RUN rustup default nightly-2020-06-05
```

Comment:
One difference here is that previously it used `nightly-2020-06-05` and the bench-build-base uses `1.42.0`. However I doubt the actual benchmarking numbers were regenerated using that nightly version :)